rviz_rendering/CMakeLists.txt

```diff
@@ -42,7 +42,7 @@ find_package(resource_retriever REQUIRED)
 # TODO(wjwwood): this block is to setup the windeployqt tool, could be removed later.
 if(Qt5_FOUND AND WIN32 AND TARGET Qt5::qmake AND NOT TARGET Qt5::windeployqt)
   get_target_property(_qt5_qmake_location Qt5::qmake IMPORTED_LOCATION)
-
+  #pragma warning(suppress : 4008)
```

Comment:
I'll note that these `#pragma` don't belong in the `CMakeLists.txt`.  Instead, they typically go into the C++ or header files.

That said, the fact that these are being introduced at all gives me pause.  Why do we need them?